### PR TITLE
Add explicit lease for 4.13 jobs

### PR DESCRIPTION
**Pull request by sgaoshang**
Add explicit lease for 4.13 jobs as well as remote worker jobs

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-vsphere-ipi-remote-worker-f14

**Comment by sgaoshang** [comment]
@jiajliu pls help review it, thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -3102,6 +3105,9 @@ tests:
         and not @fips'
       TAG_VERSION: '@4.13'
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+    leases:
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:3108]: I think need the install workflow's owner to update the job, so I guess we can leave this one untouched?

Comment (sgaoshang) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:3108]: So do you mean we do not change all agent jobs but let them change it, or let them review it?

Comment (sgaoshang) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:3108]: I see in agent disconnected job, also used   - chain: vsphere-provision-bastionhost, so I added explicit lease to all jobs with cluster_profile: vsphere-dis-2

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:3108]: okay, if that, i am okay with it.

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**